src/index.tsx

```diff
@@ -12,8 +12,13 @@ const CheckboxContext = React.createContext<CheckboxContextValue>({
   disabled: false,
 })
 
-export const useCheckbox = () =>
-  React.useContext<CheckboxContextValue>(CheckboxContext)
+export const useCheckbox = () => {
+  const context = React.useContext<CheckboxContextValue>(CheckboxContext)
+  if (context === undefined) {
```

Comment:
Please wrap this check `if (typeof process !== 'undefined' && process.env.NODE_ENV !== 'production')` to prevent it from increasing the byte count in prod